<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PackageSender</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox_netcard">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>1001</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 4px;
background: rgb(241, 246, 253);
border: 0.5px solid gray;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sport">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> 源端口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dport">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>180</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> 目的端口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_seq">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> 序号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ack">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>240</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> 确认号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ttl">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> TTL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sign">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> 标志位</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sport">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 7px;
padding: 0 10px;
background: rgb(241, 246, 253);
selection-background-color: darkgray;
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dport">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>180</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 7px;
padding: 0 10px;
background: rgb(241, 246, 253);
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="seq">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 7px;
padding: 0 10px;
background: rgb(241, 246, 253);
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ack">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 7px;
padding: 0 10px;
background: rgb(241, 246, 253);
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ttl">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 7px;
padding: 0 10px;
background: rgb(241, 246, 253);
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sign">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 7px;
padding: 0 10px;
background: rgb(241, 246, 253);
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="window">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>300</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 7px;
padding: 0 10px;
background: rgb(241, 246, 253);
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_window">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>300</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> 窗口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_data">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>550</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> 数据</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dstIP">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> 目的 IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dstIP">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>360</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 7px;
padding: 0 10px;
background: rgb(241, 246, 253);
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="srcIP">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 7px;
padding: 0 10px;
background: rgb(241, 246, 253);
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_srcIP">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> 源 IP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>540</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 14px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <activeon>blueplane.png</activeon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>350</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="tcp">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TCP</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="udp">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>UDP</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="icmp">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ICMP</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="arp">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>100</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ARP</string>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_dstMAC">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>480</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> 目的MAC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="srcMAC">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>480</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 7px;
padding: 0 10px;
background: rgb(241, 246, 253);
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dstMAC">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>480</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 7px;
padding: 0 10px;
background: rgb(241, 246, 253);
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_srcMAC">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>480</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> 源MAC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="data">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>550</y>
      <width>461</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 8px;
padding: 0 10px;
background: white;
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="terminal">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>670</y>
      <width>821</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid gray;
border-radius: 8px;
padding: 0 10px;
background: white;
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_terminal">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>660</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>  Terminal</string>
    </property>
   </widget>
   <widget class="QTableView" name="package_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1171</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="card">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 9px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <activeon>blueplane.png</activeon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>350</height>
     </size>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_sport_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>选择发送协议</string>
    </property>
   </widget>
   <zorder>comboBox_netcard</zorder>
   <zorder>label_sport</zorder>
   <zorder>label_dport</zorder>
   <zorder>label_seq</zorder>
   <zorder>label_ack</zorder>
   <zorder>label_ttl</zorder>
   <zorder>label_sign</zorder>
   <zorder>sport</zorder>
   <zorder>dport</zorder>
   <zorder>seq</zorder>
   <zorder>ack</zorder>
   <zorder>ttl</zorder>
   <zorder>sign</zorder>
   <zorder>window</zorder>
   <zorder>label_window</zorder>
   <zorder>label_data</zorder>
   <zorder>label_dstIP</zorder>
   <zorder>dstIP</zorder>
   <zorder>srcIP</zorder>
   <zorder>label_srcIP</zorder>
   <zorder>send</zorder>
   <zorder>tcp</zorder>
   <zorder>udp</zorder>
   <zorder>icmp</zorder>
   <zorder>arp</zorder>
   <zorder>label_dstMAC</zorder>
   <zorder>srcMAC</zorder>
   <zorder>dstMAC</zorder>
   <zorder>label_srcMAC</zorder>
   <zorder>data</zorder>
   <zorder>terminal</zorder>
   <zorder>label_terminal</zorder>
   <zorder>card</zorder>
   <zorder>label_sport_2</zorder>
   <zorder>package_table</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1193</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;E)</string>
    </property>
    <addaction name="action_Open"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="action_S">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
